src/pages/edit_resource/KalturaRegistration.tsx

```diff
@@ -119,38 +186,74 @@ const KalturaRegistration: FC<KalturaRegistrationProps> = ({ expanded, onChange,
-                  <>
-                    {`${t('common.showing')} ${firstItemOnPage + 1}-${lastItemOnPage} ${t('common.of').toLowerCase()} ${
-                      kalturaResources?.length
-                    }.`}
-                  </>
-                ) : (
-                  <>{t('dashboard.search_result_no_hits')}</>
-                )}
-              </Typography>
-              <Typography>{t('kaltura.choose_a_resource')}.</Typography>
-            </div>
-          )}
-          <StyledList>
+          <StyledResultList ref={startOfList}>
+            {filteredKalturaResources && kalturaResources && !busyGettingKalturaResources && (
+              <StyledFullWidth>
+                <StyledGridForFilters container justifyContent="space-between">
+                  <Grid item md={7} xs={12}>
+                    <StyledFilterBoxWrapper>
+                      <Typography display="inline" variant="body1">
+                        {t('kaltura.fill_filter_box')}:
+                      </Typography>
+                      <StyledTextFieldWithMargin
+                        onChange={(event) => setFilterValue(event.target.value)}
+                        value={filterValue}
+                        placeholder={'Filter'}
+                        variant="outlined"
+                        data-testid="filter-text-box"
+                      />
+                    </StyledFilterBoxWrapper>
+                  </Grid>
+                  <StyledCheckBoxWrapper item md={5} xs={12}>
+                    <FormControlLabel
+                      data-testid="show-already-imported-FormControlLabel"
+                      control={
+                        <Checkbox
+                          data-testid="show-already-imported-checkbox"
+                          color="default"
+                          checked={showAllResources}
+                          name="show_already_imported"
+                        />
+                      }
+                      label={t('kaltura.show_already_imported')}
+                      onChange={() => setShowAllResources(!showAllResources)}
+                    />
+                  </StyledCheckBoxWrapper>
+                </StyledGridForFilters>
+                <StyledListInfo>
+                  {filteredKalturaResources.length > 0 && (
+                    <>
+                      <Typography variant="h3" component="p" display="inline">
+                        {`${t('common.showing')} ${firstItemOnPage + 1}${
+                          lastItemOnPage !== 1 ? `-${lastItemOnPage}` : ''
+                        } ${t('common.of').toLowerCase()} ${filteredKalturaResources.length} `}
+                      </Typography>
+                      {kalturaResources.length !== filteredKalturaResources.length && (
+                        <Typography variant="body2" display="inline">
+                          {`(${kalturaResources.length - filteredKalturaResources.length} ${t('is filtered out')})`}
```

Comment:
t('is filtered out') mangler en oversettelse

Reply:
fixed